PATHFIND/GetNthClosestVehicleNodeWithHeading.md

```diff
@@ -5,11 +5,19 @@ ns: PATHFIND
 
 ```c
 // 0x80CA6A8B6C094CC4 0x7349C856
-BOOL GET_NTH_CLOSEST_VEHICLE_NODE_WITH_HEADING(float x, float y, float z, int nthClosest, Vector3* outPosition, float* heading, Any* unknown1, int unknown2, float unknown3, float unknown4);
+BOOL GET_NTH_CLOSEST_VEHICLE_NODE_WITH_HEADING(float x, float y, float z, int nthClosest, Vector3* outPosition, float* heading, int* totalLanes, int nodeType, float unknown3, float unknown4);
 ```
 
 ```
-Get the nth closest vehicle node and its heading. (unknown2 = 9, unknown3 = 3.0, unknown4 = 2.5)  
+Get the nth closest vehicle node with its heading and total lane count. (unknown3 = 3.0, unknown4 = 2.5)  
+
+Known node types: simple path/asphalt road, only asphalt road, water, under the map at always the same coords.
+The node types follows a pattern. For example, every fourth node is of the type water i.e. 3, 7, 11, 15, 19, 23, 27, 31, 35, 39... 239. Could not see any difference between nodes within certain types.
```

Comment:
Code blocks are only meant for imported natives, you should provide this as a description

Reply:
Hadn't realised the format had changed. Will find an example and fix.